### ISSUE TITLE
Request for gemeente Epe

### ISSUE THREAD

**Issue by Rincovanderbeek** [heyajohnny/afvalinfo#165]
Would it be possible to get the scraper codes for gemeente Epe?

https://mijn.circulus-berkel.nl/afvalkalender/   (personal data / house level)

**Comment by heyajohnny** [comment]
Epe is already added and should work. If not, please let me know

**Comment by Rincovanderbeek** [comment]
Thx, I am a newbee.
I’ll do my best to get this working 👍

Met vriendelijke groet,

Rinco van der Beek
Verzonden vanaf een mobiel device.

**Comment by heyajohnny** [comment]
I'm closing this issue. Please let me know if you can get it to work. Otherwise, I can help you with some example config or something.

**Comment by Rincovanderbeek** [comment]
Thanks.

I think I'll need some help.

This is what I made of it:

CONF_CITY = Epe
CONF_LOCATION = Epe
CONF_POSTCODE = 8171BC
CONF_STREET_NUMBER = 16
CONF_DATE_FORMAT = %d-%m-%Y
CONF_TIMESPAN_IN_DAYS = 365
CONF_LOCALE = NL
CONF_ID = "id"
SENSOR_PREFIX = "Afvalinfo "
ATTR_LAST_UPDATE = "last_update"
ATTR_HIDDEN = "hidden"
ATTR_IS_COLLECTION_DATE_TODAY = "is_collection_date_today"
ATTR_DAYS_UNTIL_COLLECTION_DATE = "days_until_collection_date"
ATTR_YEAR_MONTH_DAY_DATE = "year_month_day_date"

_LOGGER = logging.getLogger(__name__)

MIN_TIME_BETWEEN_UPDATES = timedelta(hours=1)



Looking at the sensors and the locations, where can I find Epe?
Or do I need to change one of these?


Sorry for bothering, trying to learn this.
Would be helpful if there also would be a programming guide somewhere.
(I only am familiar with C++).

BR,
Rinco van der Beek








Op do 7 jan. 2021 om 10:43 schreef heyajohnny <notifications@github.com>:

**Comment by heyajohnny** [comment]
You're thinking to complex... You don't need to program anything. You only need to add something like this in your 'configuration.yaml' file of Home Assistant:

```sensor:
  - platform: afvalinfo
    resources:
      - gft
      - kerstboom
      - pbd
      - papier
      - restafval
      - textiel
      - trash_type_today
      - trash_type_tomorrow
    location: epe
    postcode: 8171BC
    streetnumber: 16
    dateformat: "%d-%m-%Y"
    locale: "nl"
    timespanindays: 365
```

**Comment by heyajohnny** [comment]
Make sure u paste the code I posted in the previous comment under
'sensor:'

**Comment by Rincovanderbeek** [comment]
Thanks,

This is what I now have in my config.yaml:

sensor:
- platform: afvalinfo
Id: thuis
resources:
  - gft
  - kerstboom
  - pbd
  - papier
  - restafval
  - textiel
  - trash_type_today
  - trash_type_tomorrow
location: epe
postcode: 8171BC
streetnumber: 16
dateformat: "%d-%m-%Y"
locale: "nl"
timespanindays: 365


Unfortunately, when running a script check, it still gives some errors.
The integration is there so I don't understand where this error comes from.

[image: image.png]

One other question, when this alls works, will there then be a sensor I can
use in a lovelace card (like the schedule card?)


Kind regards,

Rinco van der Beek

Op do 7 jan. 2021 om 23:39 schreef heyajohnny <notifications@github.com>:

**Comment by Rincovanderbeek** [comment]
Hi Johnny,

I solved it, the issue was the outlining of the script.
That seems to be very critical; would be great if there is some manual how
to write these scripts :-)

I got it working, only using the Id line:
Id: thuis
Keeps giving an error so I removed it.

Thanks for for great support (and patience) !

Kind regards,

Rinco van der Beek


Op vr 8 jan. 2021 om 11:08 schreef Rinco van der Beek <
rincovanderbeek@gmail.com>:
